### DISCUSSION TITLE
How to handle tables whose schema changes regularly?

### DISCUSSION THREAD

**Discussion by jbowser-bt** [Azure/data-api-builder#2304] [Q&A] [answered]
We're experimenting with DAB as a means of providing a no-code API for database tables and views. We have one view that regularly changes based on the dynamic nature of the data. When this happens, the API call fails due to "invalid columns" that no longer exist in the view. The only workaround I've found so far is to restart DAB.

How our others handling changing schema definitions?

**Comment by seantleonard** [comment]
> We have one view that regularly changes based on the dynamic nature of the data.

Sounds like the view schema changing frequently (new, removed columns). DAB doesn't auto detect this and requires a restart currently.

**Reply by jbowser-bt** [reply]
I appreciate the response.

Do we know if support for this is on the roadmap?

**Reply by sezal98** [reply, marked as answer]
Not currently, but we will check if we can prioritize this soon. 
Resolving for now
